content/about/who-we-are.md

```diff
@@ -34,7 +34,7 @@ Thank you to all our growing list of contributors who have made Quire what it is
 - [@egardner](https://github.com/egardner)
 - [@fchasen](https://github.com/fchasen)
 - [@garrettdashnelson](https://github.com/garrettdashnelson)
-- @ggarcia0596
+- [@ggarcia0596](https://github.com/ggarcia0596)
```

Comment:
This became relinked again

Reply:
Yes, I relinked it because it looked weird being the only name without a hyperlink. I figured the reality of someone going through and clicking all those links is pretty slim.

Reply:
Ah ha! fair enough